lib/static/new-ui/features/suites/components/SuitesTreeView/utils.ts

```diff
@@ -316,12 +358,32 @@ export const sortTreeNodes = (entitiesContext: EntitiesContext, treeNodes: TreeN
                 if (weight.metadata.failedRunsCount !== undefined) {
                     newAccWeight.metadata.failedRunsCount = (newAccWeight.metadata.failedRunsCount ?? 0) + weight.metadata.failedRunsCount;
                 }
+                if (weight.metadata.duration !== undefined) {
+                    newAccWeight.metadata.duration = (newAccWeight.metadata.duration ?? 0) + weight.metadata.duration;
+                }
 
                 return newAccWeight;
             }, createWeight(new Array(weights[0]?.value?.length)));
         }
 
-        return createWeight([0]);
+        if (currentSortExpression.type === SortType.ByStartTime) {
+            const MAX_DATE = new Date(8640000000000000).getTime();
```

Comment:
magic value `8640000000000000`
We could define it separately as `MAX_TIMESTAMP`

Btw, why aren't you using just `const MAX_TIMESTAMP = 8640000000000000`? `new Date(N).getTime()` returns `N` for valid timestamps, and here the timestamp is defined, so there is no need for `MAX_DATE`, `MAX_TIMESTAMP` would be enough